study.md

```diff
@@ -34,7 +34,11 @@ including class. Why might Array redefine methods included from Enumerable?
 Please give reasons for the methods you list.
 
 ```md
-<!-- your answer here -->
+
+One shared instance method is select.  For an array, a new array is returned and a block returns a true value.  For an enumerable, the select instance method operates on a hash instance.  Although select acts similarly for both arrays and enumerables, it will return a hash object when it is operated on by a hash.
+
+Another shared instance method is reject.  For both arrays and enumerables, the reject instance method reutrns an array.  However, it appears that the array instance method returns a new array at the end, rather than the original array.
+
```

Comment:
`Array` might redefine methods included from `Enumerable` if custom behavior is desired.  For example, `Array` might redefine `each` to only return elements with even indices on each iteration.